src/content/about/institutions.mdx

```diff
@@ -0,0 +1,42 @@
+import { MaxWidthWrapper } from "../../components/layout/maxWidthWrapper";
+import {
+  InstitutionList,
+  InstitutionItem,
+} from "../../components/sections/institutionList";
+import NSF from "../../images/about/nsf-logo.svg";
+import StateOfNC from "../../images/about/state-nc-seal-logo.svg";
+import NCPolicyCollaboratory from "../../images/about/policy-collab-logo.png";
+import NHLBI from "../../images/about/nhlbi-logo.svg";
+import NIH from "../../images/about/nih-logo.png";
+
+<MaxWidthWrapper backgroundColor="#F5F6F5" gutterBottom>
+
+## HeLx is made possible through projects supported by the following funders
```

Comment:
overall this file looks great! for this title here, you may need to use a `<HeadingSection/>` component to get the title centered, and then add a `maxWidth="lg"` since the default is `"sm"`